client/coral-admin/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "build": "./node_modules/.bin/webpack --config webpack.config.js",
+    "build": "npm install && ./node_modules/.bin/webpack --config webpack.config.js",
```

Comment:
`npm install` is being run at the root level: https://github.com/coralproject/talk/blob/master/package.json#L8